src/ziggurat/metrics.clj

```diff
@@ -1,20 +1,20 @@
 (ns ziggurat.metrics
-  (:require [clojure.tools.logging :as log]
-            [clojure.walk :refer [stringify-keys]])
+  (:require [clojure.string :as str]
```

Comment:
you have added str as a dependency, but I cannot see it being used anywhere.

Reply:
its on (intercalate-dot) function, I remember it was something like (apply str (interpose...)). Maybe somone changed it before getting merged?

Reply:
oh geeez, its different in 2.x and master...
now Im confused...

Reply:
removed str dependency, done...